static/src/javascripts/projects/common/modules/discussion/user-edit-link.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import config from 'lib/config';
+import fastdom from 'lib/fastdom-promise';
+
+const addEditLink = (containerEl: HTMLElement): void => {
+    const myUserId: string = config.get('user.id');
+    const pageUserId: string = containerEl.dataset.userid;
+
+    const parentNode: ?Element = containerEl.parentElement;
+
+    if (parentNode && myUserId === pageUserId) {
```

Comment:
what if both `myUserId` and `pageUserId` are `undefined`?

Reply:
ah, good point!